packages/oid4vci-demo-frontend/src/configs/banqup-identity.json

```diff
@@ -0,0 +1,249 @@
+{
+  "general": {
+    "credentialName": "Identity credential",
+    "issueCredentialType": "VerifiedID",
+    "oid4vpAgentBaseUrl": "https://identity.banqup.demo.sphereon.com/agent",
+    "oid4vciAgentBaseUrl": "https://identity.banqup.demo.sphereon.com/agent",
+    "verifierUrl": "https://sphereon.com",
```

Comment:
think we need to make this match the banqup verifier

Reply:
hmm, i now see this is used on some button. have not looked at what it says in the design and if the button is even used

Reply:
If memory serves me correctly this was the button on the last page of the flow, pointing to our siopv2 verifier demo with the red/white/black design. Since then the issuer demo evolved, so it was also used to go to other places in the last page. Now in the multi card demo's we are automatically redirected to the landing page. So guess the name needs to be adjusted for the button.

Reply:
I removed it from the banqup configs. Testing DPP-84 wil reveal if all the other demo's are still ok